src/overlays/actors/ovl_En_Trt/z_en_trt.h

```diff
@@ -2,12 +2,64 @@
 #define Z_EN_TRT_H
 
 #include <global.h>
+#include "overlays/actors/ovl_En_GirlA/z_en_girla.h"
 
 struct EnTrt;
 
+typedef void (*EnTrtActionFunc)(struct EnTrt*, GlobalContext*);
+typedef void (*EnTrtBlinkFunc)(struct EnTrt*);
+
 typedef struct EnTrt {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x2EC];
+    /* 0x144 */ EnTrtActionFunc actionFunc;
+    /* 0x148 */ EnTrtActionFunc tmpActionFunc;
+    /* 0x14C */ SkelAnime skelAnime;
+    /* 0x190 */ ColliderCylinder collider;
+    /* 0x1DC */ UNK_TYPE1 pad1DC[0x154];
+    /* 0x330 */ s16 delayTimer;
+    /* 0x332 */ s16 sleepSoundTimer;
+    /* 0x334 */ s16 cutsceneState;
+    /* 0x338 */ s32 stickAccumX;
+    /* 0x33C */ s32 stickAccumY;
```

Comment:
Are these the accumulated/threshold of how far you held in the x and y direction?  just trying to make sense of the name.

Reply:
This is the accumulated distance held in the x and y direction. Logic for it is in `EnTrt_UpdateJoystickInputState`, but basically it accumulates every frame up to a min/max of +/-2000, unless you are within +/-30 or changing stick direction.